core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -622,6 +623,12 @@ public String toString() {
 
     @Override
     public String toStringRequestOnly() {
+        return toStringRequestOnly(Function.identity(), Function.identity());
+    }
+
+    @Override
+    public String toStringRequestOnly(Function<HttpHeaders, HttpHeaders> headersSanitizer,
+                                      Function<Object, Object> contentSanitizer) {
```

Comment:
I picked the types to match the types of e.g. `requestHeaders()`, `requestContent()` - I can't think of any case we need `super` and it makes the signature less clear to me, similarly since the content types are `Object`, `?` seems less clear to me. Let me know what you think though.